src/axes.jl

```diff
@@ -504,17 +504,14 @@ end
 const _widen_seriestypes = (:line, :path, :steppre, :stepmid, :steppost, :sticks, :scatter, :barbins, :barhist, :histogram, :scatterbins, :scatterhist, :stepbins, :stephist, :bins2d, :histogram2d, :bar, :shape, :path3d, :scatter3d)
 
 function default_should_widen(axis::Axis)
-    should_widen = false
-    if !(is_2tuple(axis[:lims]) || axis[:lims] == :round)
```

Comment:
I do think that we want to keep `:round` having precedence here

Reply:
Really? I think the result of `ylim=(0,1)` and `ylim=:round` should be the same if the data fall within 0 < y < 1. I would rather take the widen=:force route than to have different behavior for different ways of specifying the limits.

FWIW, `ylim=:round` has no effect for me (on gr but I don't think it should matter), i.e. the axis just ends wherever the data ends.

Reply:
Here is one example for `round` vs `widen`
```julia
julia> plot([0.1,0.5,3.0,3.6], rand(4)) |> Plots.xlims
(-0.0049999999999999906, 3.705)

julia> plot([0.1,0.5,3.0,3.6], rand(4), xlims = :round) |> Plots.xlims
(0.0, 4.0)
```